.mergify.yml

```diff
@@ -21,11 +13,37 @@ pull_request_rules:
       - -merged
       - -closed
       - author!=dependabot[bot]
-    actions:
-      queue:
-        method: squash
-        name: default
+    merge_conditions:
+      # Conditions to get out of the queue (= merged)
+      - status-success=Run Unit Tests
+      - status-success=Semantic Pull Request
+      - status-success=Analyze (javascript)
+    merge_method: squash
+  - name: duplicated default from Automatically approve and merge Dependabot PRs
+    queue_conditions:
```

Comment:
again, duplicated from `pull_request_rules -> conditions` for the condition named "Automatically approve and merge Dependabot PRs"